ogcapi-features/README.md

```diff
@@ -110,6 +112,88 @@ STAC API.
 Implementing OAFeat enables a wider range of clients to access the API's STAC Item objects, as it is a more widely implemented
 protocol than STAC. 
 
+## Item Pagination
+
+OAFeat supports paging through hypermedia links for the Items resource 
+(`/collections/{collectionId}/items`). Since only GET requests are allowed for this endpoint, 
+it is possible
+to provide a Link with `rel` type `next` and the href of the full URL of the next page of results.
+This link href must contain any URL parameters that are necessary 
+for the implementation to understand how to provide the next page of results, eg: `page`, `next`,
+`token`, etc. For example, the links array will have an object like:
+
+```json
+    "links": [
+        {
+            "rel": "next",
+            "href": "http://api.cool-sat.com/collections/my_collection/items?page=2"
+        }
+    ]
+```
+
+The href may contain any arbitrary URL parameter, which is implementation-specific:
+
+- `http://api.cool-sat.com/collections/my_collection/items?page=2`
+- `http://api.cool-sat.com/collections/my_collection/items?next=8a35eba9c`
+- `http://api.cool-sat.com/collections/my_collection/items?token=f32890a0bdb09ac3`
+
+Additionally, STAC has extended the Link object to support additional fields that provide header values
+to the client should they be needed for a subsequent request for the next page of results. The use
+of header values for pagination with GET requests is uncommon, so if your implementation does not use them you can
+omit this attribute in the Link. These
+fields are described in detail in the [Item Search](../item-search/README.md#pagination) spec.  To avoid returning
+the entire original request in the response, the  `merge` 
+property can be specified. This indicates that the client must send the same request headers that were sent in the original 
+request, but with the specified headers values merged in. This allows servers to indicate what needs to change 
+to get to the next page without mirroring the entire request back to the client.
+
+Implementations may also add link relations `first` and `last`, though these are not required and may
+be infeasible to implement in some data stores.
```

Comment:
I understand that `last` may be hard to implement, but `first` should be pretty easy. You just need to add the "initial" link again, without any pagination parameters.

Reply:
good point, i'll rewrite that.

Reply:
rewritten.